src/vs/workbench/services/files/electron-browser/fileService.ts

```diff
@@ -168,6 +169,10 @@ export class FileService implements IFileService {
 		return this.raw.resolveContent(resource, options);
 	}
 
+	public resolveStringStream(resource: uri, options?: IResolveContentOptions, token?: CancellationToken): IStringStream {
```

Comment:
What does this new method mean for the remote file system implementation? Can a remote file system provide the same stream, I have not seen any updates of this for [`RemoteFileService`](https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/services/files/electron-browser/remoteFileService.ts#L79)

Reply:
In parts it actually already works like that. Tho, the 'old' stream function does do everything in once call. My plan is to nuke the old once this is done and then migrate the remote work. Makes sense?

Reply:
Yes.